src/TextUI/TestRunner.php

```diff
@@ -341,77 +341,71 @@ public function run(TestSuite $suite, array $arguments = [], array $warnings = [
         $coverageFilterFromOption            = false;
         $codeCoverageReports                 = 0;
 
-        if (!isset($arguments['noLogging'])) {
```

Comment:
It's worth viewing TestRunner changes with whitespace changes hidden:
https://github.com/sebastianbergmann/phpunit/pull/4446/files?w=1